fits-loader-docker/sqs-process.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+
+RAW_MESSAGE=$(aws sqs receive-message --wait-time-seconds 20 --visibility-timeout 300 --queue-url "${SQS_QUEUE_URL}")
+
+if [[ "${RAW_MESSAGE}" != "" ]]; then
```

Comment:
maybe

`if  [[ ! -z "${RAW_MESSAGE}"  ]; then`

but it's the same thing and I think the full version is more readable.